sklearn/cluster/k_means_.py

```diff
@@ -623,7 +638,7 @@ def _labels_inertia(X, x_squared_norms, centers,
 
 
 def _init_centroids(X, k, init, random_state=None, x_squared_norms=None,
-                    init_size=None):
+                    init_size=None, check_inputs=True):
```

Comment:
Its passed to _k_init (the kmeans++) implementation. On reviewing this again it may not be worth the extra API complexity to have the check_inputs parameter in _init_centroids because it is not called very many times.